### PR TITLE
Fix typo: remove comma

### PR DESCRIPTION
**Pull request by mrm8488**

### CODE REVIEW

**Review by osanseviero — COMMENTED** [review]
Actually it's missing I think? cc @merveenoyan

**Comment by mrm8488** [comment]
> Actually it's missing I think? cc @merveenoyan

Yup, sorry it is missing!

File: paligemma.md [hunk 1]
```diff
@@ -127,7 +127,7 @@ model = PaliGemmaForConditionalGeneration.from_pretrained(model_id)
 processor = AutoProcessor.from_pretrained(model_id)
 
 prompt = "What is on the flower?"
-image_file = https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/bee.jpg?download=true"
+image_file = https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/bee.jpg?download=true
```
Comment (merveenoyan) [inline paligemma.md:130]:
```suggestion
image_file = "https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/bee.jpg?download=true"
```

**Review by merveenoyan — APPROVED** [review]
thanks!

**Pull request merged**